src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/CSharpPassSystemUriObjectsInsteadOfStrings.cs

```diff
@@ -13,10 +13,10 @@ namespace Microsoft.CodeQuality.CSharp.Analyzers.ApiDesignGuidelines
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class CSharpPassSystemUriObjectsInsteadOfStringsAnalyzer : PassSystemUriObjectsInsteadOfStringsAnalyzer
     {
-        protected override SyntaxNode? GetInvocationExpression(SyntaxNode node)
+        protected override SyntaxNode? GetInvocationExpression(SyntaxNode invocationNode)
```

Comment:
I don't see fixes to []IDE0090](https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0090) in this PR. Can you please add them?

Reply:
Also, I think we should bump up the CodeStyle layer version as part of the PR: https://github.com/dotnet/roslyn-analyzers/blob/4a6dae86ccb74fd045c89c53f394cc30de068fcb/eng/Versions.props#L26

This will ensure that these new analyzers are also enforced on build, and prevent future violations. Moving to newer CodeStyle layer might require you to bump the underlying compiler toolset version, similar to https://github.com/dotnet/roslyn-analyzers/pull/4449

Reply:
It was not available with current version. I will make this change and try again.

Reply:
@Evangelink If it is too much trouble, it is fine to do it later whenever this need comes up. I am fine with your current PR, I'll let @dotpaul review/approve/merge as the owner and maintained of this branch.